src/participant/constants.ts

```diff
@@ -16,6 +16,7 @@ export type ParticipantResponseType =
 interface Metadata {
   intent: Exclude<ParticipantResponseType, 'askForNamespace'>;
   chatId: string;
+  docsChatbotMessageId?: string;
```

Comment:
There is more than one message per chatId, and we need to return messageId with each response to know which exact message a user rates.